src/pages/index.js

```diff
@@ -9,6 +10,30 @@ import Section from "../components/Section.js";
 import UserInfo from "../components/UserInfo.js";
 import * as utilities from "../utility/utility.js";
 import * as constants from "../utility/constants.js";
+const api = new API({
+  baseUrl: constants.config.baseUrl,
+  headers: constants.config.headers,
+});
+let cardRenderer;
+let cardArray;
+api.getUserInfo().then((result) => {
+  userInfo.setUserInfo(result);
+  constants.avatar.src = result.avatar;
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**